### DISCUSSION TITLE
I have an M5 dial, I burned a project and I can no longer see the COM8 port I need a light!

### DISCUSSION THREAD

**Discussion by aliendeltaorion** [community/community#103270] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hi everyone, I downloaded a project here from this link for my M5 Dial, I can no longer get the device to recognize the Serial port, the project is a mouse simulator, but after installing the project my driver became ITE USB CONNECTOR CLIENT, I don't know how to reverse this, I've already tried using ESPTOOL to flash an original .bin, but as the device doesn't appear on the serial port, how can I flash it? or revert the installation of this project? this is the project link: https://github.com/webgyver/M5Dial-Arduino-Tabbing-Navigation-HID
I would be very grateful if someone can help me! thank you very much!
![IMG_2206](https://github.com/community/community/assets/157915651/4f9ab823-6d55-4f96-9edd-c923f85aabd9)
![IMG_2207](https://github.com/community/community/assets/157915651/9ebbe5a2-025d-42bd-a32e-807c057977ea)

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.
